modules/arm_plugin/scripts/arm_cpu_plugin_build.sh

```diff
@@ -127,6 +131,7 @@ cmake -DOpenCV_DIR=$STAGING_DIR/extras/opencv/cmake -DENABLE_OPENCV=OFF \
       -DPYTHON_INCLUDE_DIRS="/opt/python3.7_arm/include/python3.7m" \
       -DPYTHON_LIBRARY="/opt/python3.7_arm/lib/libpython3.7m.so" \
       -DENABLE_PYTHON=ON \
```

Comment:
should we disable the python at all? since it's built later once again, but together with wheel.

Reply:
I don't know if it is necessary, I tried to follow compilation process on the AZURE CI: https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux_arm64.yml#:~:text=%2D%20task%3A,python%20binding%27